components/workflow-mgt/org.wso2.carbon.identity.workflow.impl/src/main/java/org/wso2/carbon/identity/workflow/impl/internal/WorkflowImplServiceComponent.java

```diff
@@ -201,4 +209,18 @@ protected void setIdentityCoreInitializedEventService(IdentityCoreInitializedEve
          is started */
     }
 
+    protected void setWorkflowImplServiceListener(WorkflowImplServiceListener workflowListener) {
+        if(workflowListener !=null) {
```

Comment:
Do we need null check here ? I don't think anyone can register a service with null object